src/pages/RepoPage/ATSTab/ATSNotConfigured.tsx

```diff
@@ -0,0 +1,40 @@
+import Button from 'ui/Button'
+
+function ATSNotConfigured() {
+  return (
+    <div className="flex flex-col justify-evenly space-y-7 bg-ds-gray-primary px-4 py-6">
+      <h1 className="mb-2 text-xl font-semibold">
+        Automated Test Selection Not Yet Configured
+      </h1>
+      <h2 className="mb-2 text-lg font-semibold">
+        {`\u{1F680}`} Ready to Accelerate Your CI/CD Pipeline?
+      </h2>
+      <div>
+        <p className="mb-0">
+          Automated Test Selection is waiting to be set up!
+        </p>
+        <ul className="list-disc pl-6">
```

Comment:
I feel these are a bit close to one another, is this part of the designs?

Reply:
Yeah, linked in the description.

Reply:
Looking at the designs, I'm feeling perhaps these have a gap-0.5 or a gap-1, wdyt?

Reply:
Changed to `gap-1`.

Reply:
I changed the spacing to `gap-1` for the outer container. Don't think changing the spacing between list elements will make a meaningful difference.